classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -30,7 +30,7 @@ class WebserviceSpecificManagementImagesCore implements WebserviceSpecificManage
     /** @var string */
     protected $output;
 
-    /** @var WebserviceRequest */
+    /** @var WebserviceRequestCore */
```

Comment:
Same here

Reply:
That's a no go for me, I'm gonna block the PR for this problem because we shouldn't let a static tool dictate us the use of some classes that are absolutely not meant to be exposed

Reply:
Check above